CMakeLists.txt

```diff
@@ -64,7 +64,7 @@ endif()
 
 # RCP platform library
 if(NOT OT_PLATFORM_LIB_RCP)
-    set(OT_PLATFORM_LIB_RCP "openthread-efr32-rcp" CACHE STRING "default efr32 RCP platform library" FORCE)
+    set(OT_PLATFORM_LIB_RCP "openthread-efr32-rcp-uart" CACHE STRING "default efr32 RCP platform library" FORCE)
```

Comment:
Do we need a `openthread-efr32-rcp-spi` library or should this remain as `openthread-efr32-rcp` and its used for both spi and uart?

Reply:
Actually I see this comment in `script/build` 
``` 
# ==============================================================================================================
# Generate openthread-efr32-rcp-spi and openthread-efr32-rcp-spi-mbedtls libs
# ==============================================================================================================
```
So don't we need `openthread-efr32-rcp-spi`.

Reply:
Same comment applies here too.